main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
+Run the `corepack enable` command.
+`corepack enable`
 
-Leave this process and its logs running in its own terminal window.
+![023](./assets/023.png)
 
-## Open the Agoric Wallet and REPL
+Now run the `yarn –version` command
+`yarn –version`
 
-```sh secondary style2
-# Terminal 2
-cd demo
-agoric open --repl
+![049](./assets/049.png)
+
+
+# Installing Docker
+Now you'll install Docker using the two commands below. This first command will add the Docker GPG keys to your system, then add the repository to Apt for installation.
+```
+# Install Docker
+# Add Docker's official GPG key:
+sudo apt-get update
+sudo apt-get install ca-certificates curl gnupg
+sudo install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+sudo chmod a+r /etc/apt/keyrings/docker.gpg
+
+# Add the repository to Apt sources:
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
 ```
+![027](./assets/027.png)
 
-This should automatically open [http://127.0.0.1:8000](http://127.0.0.1:8000) in a new browser window or tab.
+Now you can install Docker!
+`sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin`
 
-To begin using the wallet, click the "Connect Solo Wallet" button.
+![028](./assets/028.png)
 
-![agoric wallet solo connection](./assets/agoric-open-repl-1.png)
+Now that Docker has been installed you'll need to add your user account to the Docker group.
+`sudo usermod -aG docker $USER`
 
-After your solo wallet is connected, then you're ready to deploy the contract and API. 
+![029](./assets/029.png)
 
-## Deploy the Contract and API
+Now go ahead and reboot your image.
 
-In our second terminal, deploy the contract to the simulated blockchain
-and the API to the solo client.
+![030](./assets/030.png)
 
-```sh secondary style2
-# Terminal 2
-cd demo # if not already there
-agoric deploy ./contract/deploy.js 
-agoric deploy ./api/deploy.js
-```
+Once your image has rebooted and you've logged back on, test that Docker works by running the `hello-world` sample.
+`docker run hello-world`
 
-We'll cover [deploying smart contracts](./deploying.md)
-in detail later.
+![031](./assets/031.png)
 
-## Start the Dapp User Interface
 
-The web user interface communicates with the API in the solo client as well as the wallet.
+# Installing the Sample Dapp
+Now you'll use yarn to pull down the sample dapp. The sample dapp will be placed in a subfolder named `demo`.
+`yarn create @agoric/dapp --dapp-base https://github.com/agoric-labs/ --dapp-template dapp-game-places demo`
 
-```sh secondary style3
-# Terminal 3
-cd demo # if not already there
-cd ui && yarn start
-```
+![024](./assets/024.png)
+
+
+# Install Dapp Dependencies
```

Comment:
In #881, I included some motivation, since this step is so burdensome:

"The UI depends on the React framework, and the contract depends on the Agoric framework. The packages in this project also have development dependencies for testing, code formatting, and static analysis."

Reply:
I also included a troubleshooting details box:

https://github.com/Agoric/documentation/blob/312d426b33d1f5be23d6d40fc980b2d801728390/main/guides/getting-started/README.md?plain=1#L122-L132

Reply:
Added